paasta_tools/paastakube/api_client.py

```diff
@@ -0,0 +1,684 @@
+"""
```

Comment:
Can you leave a comment around the bit you've fixed and a link to the GH issue(s)

Also worth adding the same comment to this line I think: `self.deployments = kube_client.AppsV1Api(ApiClient())`